web/vtadmin/src/proto/vtadmin.d.ts

```diff
@@ -14536,6 +14536,9 @@ export namespace tabletmanagerdata {
 
         /** GetSchemaRequest exclude_tables */
         exclude_tables?: (string[]|null);
+
+        /** GetSchemaRequest table_schema_only */
+        table_schema_only?: (boolean|null);
```

Comment:
These changes have nothing to do with my new line change. I thought we had a workflow to test that vtadmin files are upto-date on the PR being merged. How did this happen? @ajm188

Reply:
my best guess: in #10552 @shlomi-noach ran the generators (and CI passed on his branch). separately, you ran the generators (and CI passed on _your_ branch), but because these two changes didn't step on each other, git happily merged the latter (yours) without making you rebase. contrast this with `protoc-gen-go` which includes that magic byte array that always forces a conflict

Reply:
well, even simpler explanation, it turns out: 
<img width="828" alt="Screen Shot 2022-06-22 at 6 17 01 AM" src="https://user-images.githubusercontent.com/4276638/175005640-a786040f-55f4-4469-8686-aa0933dd21b7.png">